src/d3d9/d3d9_device.h

```diff
@@ -1241,6 +1241,18 @@ namespace dxvk {
 
     uint64_t GetCurrentSequenceNumber();
 
+    D3D9SwapChainEx* GetLastUsedSwapchain() {
+      return m_mostRecentlyUsedSwapchain;
+    }
+
+    void SetLastUsedSwapchain(D3D9SwapChainEx* swapchain) {
+      m_mostRecentlyUsedSwapchain = swapchain;
+    }
+
+    void ResetLastUsedSwapchain() {
```

Comment:
Consider renaming this to `RefreshLastUsedSwapchain()`

Reply:
But it doesn't refresh it, it resets it to the built-in swapchain.